aptos-move/framework/aptos-framework/sources/transaction_validation.move

```diff
@@ -327,6 +387,11 @@ module aptos_framework::transaction_validation {
 
         // Increment sequence number
         let addr = signer::address_of(&account);
-        account::increment_sequence_number(addr);
+        if (!account::account_resource_exists_at(addr) && features::lite_account_enabled()) {
+            lite_account::increment_sequence_number(addr);
+        } else {
+            // account v1
+            account::increment_sequence_number(addr);
```

Comment:
What if `account::account_resource_exists_at(addr)` is false, but feature is enabled? Should we be considering three cases: V1 exists, lite account, V1 doesn't but lit account is also not enabled?

Reply:
if feature is enabled, and resource does not exist, lite account exists implicitly as the address itself will be taken as auth key.